src/behaviour.rs

```diff
@@ -946,10 +989,10 @@ mod tests {
         let cid: Cid = "QmP8njGuyiw9cjkhwHD9nZhyBTHufXFanAvZgcy9xYoWiB"
             .parse()
             .unwrap();
-        let peer_id: PeerId = "12D3KooWC1EaEEpghwnPdd89LaPTKEweD1PRLz4aRBkJEA9UiUuS"
+        let peer_id: PeerId = "QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ"
             .parse()
             .unwrap();
-        let multiaddr: Multiaddr = "/ip4/95.217.194.97/tcp/8008".parse().unwrap();
+        let multiaddr: Multiaddr = "/ip4/104.131.131.82/tcp/4001".parse().unwrap();
```

Comment:
this is always such a pain — can you guarantee that this address will keep working?

Reply:
I cannot guarantee it will keep working but it has been updated and have been stable so far in use. I recall updating it due to the previous address causing the test to fail.